gateway/ln-gateway/src/rpc/rpc_server.rs

```diff
@@ -31,9 +32,16 @@ pub async fn run_webserver(
         .route("/connect", post(connect))
         .layer(RequireAuthorizationLayer::bearer(&authkey));
 
+    // TODO: Provide admin UI path from configurations
+    let admin_ui_path = Path::new("../../../admin-ui/build");
+
+    // Serve admin UI
+    let admin_ui = SpaRouter::new("/", admin_ui_path);
```

Comment:
this is really neat!

Reply:
Not sure we'll go with this, given the need for bundlng UI files with the binary